login-fire-social.html

```diff
@@ -66,24 +67,26 @@
       }
     </style>
 
-    <template is="dom-if" if="[[onlySignOut]]">
+
+
+    <template is="dom-if" if="[[_showOneButton]]">
```

Comment:
Yeah, I know that we can use a bundler or Vulcanize to handle this but my concern was whether not using one would break things. Added #90